Packs/SplunkPy/ReleaseNotes/1_3_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### SplunkPy
+- Fixed an issue where some events were dropped, instead of being properly fetched, while taking edge cases into account.
```

Comment:
```suggestion
- Fixed an issue where some events were dropped instead of being properly fetched
- Improved fetch resiliency by addressing certain edge cases.
```